virttest/qemu_vm.py

```diff
@@ -1509,6 +1509,25 @@ def sort_key(dev):
         add_memorys(devices, params)
         mem = int(params.get("mem", 0))
 
+        # Get cpu model, before add smp, to determine cpu topology
+        cpu_model = params.get("cpu_model", "")
+        use_default_cpu_model = True
+        if cpu_model:
+            use_default_cpu_model = False
+            for model in re.split(",", cpu_model):
+                model = model.strip()
+                if model not in support_cpu_model:
+                    continue
+                cpu_model = model
+                break
+            else:
+                cpu_model = model
```

Comment:
suggest to set  cpu_mode  to default cpu module here.

Reply:
@xutian Thanks for your review. It's moved from Line 1649 and I don't like to modify it in this patch. And Line 1529 has set to default cpu model too.